lib/search.js

```diff
@@ -15,7 +15,8 @@ function search(root, expression, settings) {
   var depth = null
   var lookingForToc = expression !== null
   var maxDepth = settings.maxDepth || 6
-  var parents = settings.parents || []
+  var parents =
+    typeof settings.parents === 'undefined' ? 'root' : settings.parents
```

Comment:
Would `settings.parents || 'root'` do the job?

Reply:
I was thinking of explicitly passing in `null` option as `settings.parents`, probably no need to replace `null` with default option at this point?

Reply:
It’s what we do in other places as well, so I’d go with the current style, `settings.parents || 'root'`!

Reply:
Note: see below comment